egs/cifar/v1/image/ocr/make_features.py

```diff
@@ -88,10 +88,16 @@ def horizontal_pad(im, allowed_lengths = None):
         left_padding = int(padding // 2)
         right_padding = padding - left_padding
     dim_y = im.shape[0] # height
-    im_pad = np.concatenate((255 * np.ones((dim_y, left_padding, args.num_channels),
-                                           dtype=int), im), axis=1)
-    im_pad1 = np.concatenate((im_pad, 255 * np.ones((dim_y, right_padding, args.num_channels),
-                                                    dtype=int)), axis=1)
+    if args.num_channels == 4:
```

Comment:
I guess, it is an existing issue, we can update the if condition to if args.num_channels == 1 or args.num_channels == 4:, it will fix it for num_channels = 1 as well